client/src/components/dashboard/dashboardSettings/dashboardSettings.tsx

```diff
@@ -1,14 +1,50 @@
+import React from "react";
 import useAppStore from "../../../stores/zustand/appStore";
 
 const DashboardSettings = () => {
-    const data = useAppStore((state) => ({
-        vendorName: state.vendor.fullname.trim(),
-        vendorEmail: state.vendor.email,
-    }));
-    console.log(data)
+
+    let appState = useAppStore.getState();
+    let [vendor, setVendor] = React.useState({ ...appState?.vendor });
+
+
+    let handleOnChange = (e: any, key: string) => {
```

Comment:
@GitUserVivek

- Don't use any, set the event type to "(e: React.ChangeEvent<HTMLInputElement>)"
- Use name attribute on form input elements to know about the source